mobile/lib/features/student_management/presentation/pages/home_page.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import '../widgets/drawer_overlay.dart';
+
```

Comment:
Why is there another home_page?

Reply:
Because it was separeted on class and user so i kept it like that on student, but i am planningo on creating only one file with aaaaaall routes e a single home page as well, that answers the next comment

Reply:
fixed that on other branch